intro_calculating_cred_grain.md

```diff
@@ -0,0 +1,162 @@
+
+-   a contributor should never lose cred. They ought to become more diluted as other contributors continue to earn new cred, but they’ll still continue to earn Grain on that Cred.
+-   a past action that was recently found to be valuable in the present should cause new Cred to flow to that action.
+
+#### Time epochs and fibration
+
+To simplify the computational overhead and storage required to compute Cred over time, SourceCred is moving toward a fibration model where a node is added per contributor, per unit time (currently weekly for SourceCred). In this model, epoch nodes intermediate flows to a user from any nodes during a period of time, and have a fixed transition probability to the user’s node, rather than a weight from which transition probabilities are calculated. The remaining transition probability is then shared between the seed node (for the teleportation vector) and the other node weights. This will be implemented by distinguishing graph edges, which have fixed weights, from markov edges, which have fixed transition probabilities from epoch nodes. This is then input into the mechanism that calculates the transition probabilities.
+
+## Grain dynamics
+
+#### How Grain is generated
+
+Once SourceCred calculates the Cred for individual contributors, that Cred is linked to an identity that is an amalgamation of their identities across the platforms (e.g. GitHub handle or Discord username). A constant amount of SourceCred Grain (¤) is produced per unit time, currently 15000¤ per week. This amount is distributed according to two different distribution mechanisms.
+
+-   20% of the grain is “fast,” and is disbursed to nodes in proportion to the cred earned by each node in the past week. This is intended to reward new participants.
+-   80% of the grain is “slow,” and is disbursed in proportion to the difference between total grain you've been distributed, and what that node’s proportional fair Cred would be over all history
+
+As a result, if you didn't do anything this week, you have only slow Grain. Alternatively, if you joined this week, you’ll only have fast Grain. The fast component can potentially over-distribute grain, but it will get diluted by other contributions.
+
+#### How Grain is traded:
+
+Grain payouts are currently tracked by Dandelion in an observable notebook linked to in [this Discourse thread](https://discourse.sourcecred.io/t/sourcecred-contributor-payouts/298). Additionally, Grain distributions will vest over time in order to smooth out the peakiness of grain distribution, as described in [this Discourse thread](https://discourse.sourcecred.io/t/grain-vesting/636). Even now, Grain can be sold or given. In fact, [Protocol Labs](https://protocol.ai/) supports SourceCred by purchasing Grain directly from contributors. At present, the transfer of Grain is executed by recording changes to the observable notebook on GitHub [here](https://github.com/sourcecred/cred).
```

Comment:
Mention that we currently consider grain a "preview" or experimental balance, though we plan to honor it going forward.

Reply:
I've never seen the "Outdated" tag on github comments; is that something you applied, or is this a change you'd still like to see?

Reply:
Outdated gets applied automatically when a line that was commented on has a commit that changes it. It doesn’t necessarily indicate that the comment is irrelevant, most mostly a heads up to the reader that it might be.

Reply:
The succinct version: https://github.blog/changelog/2018-10-30-outdated-comments

I remember when they used to be collapsed automatically and it made everything really confusing. They used to do this accordion thing with the entire thread too and it would make me miss content in PRs regularly. Thankfully I have t seen that for a while! But TLDR outdated != no longer relevant, but just changed.

Reply:
Very good to know.  I've now made this change, as it's very much still relevant.  Thanks for the explanation @vsoch!